specs/data-flow-and-verification.md

```diff
@@ -11,32 +11,31 @@ the exit point is execution by a rollup node.
 
 ## Entry point
 
-The entry point for rollup data is via a `sequence::Action`, which can become
-part of a sequencer transaction. The data types are as follows:
+The entry point for rollup data is via a `RollupDataSubmission` action, which can
+become part of a sequencer transaction. The data types are as follows:
 
 ```rust
-// sequence::Action
-pub struct Action {
-    pub(crate) chain_id: Vec<u8>,
-    pub(crate) data: Vec<u8>,
+pub struct RollupDataSubmission {
+    rollup_id: Vec<u8>,
+    data: Vec<u8>,
 }
```

Comment:
It looks more like this now:
```rust
pub struct RollupDataSubmission {
    rollup_id: RollupId,
    data: Bytes,
    fee_asset: asset::Denom,
}
```
There probably should be a short explanation of the `fee_asset` field below where we already describe `rollup_id` and `data`?